chamber-of-commerce.md

```diff
@@ -0,0 +1,96 @@
+3.4 Proposals are reviewed by the Chamber of Commerce Board of Directors.
+
+### 4. Post-Approval Steps
+4.1 Upon approval, individuals must complete the rental process for their designated commercial plot within one week.
+4.2 Individuals must move into their business within one week of renting the plot. 
+4.3 Businesses must be open for business within one month of the approval of their business.
+4.4 Businesses must adhere to all sim rules and regulations, including any specific to commercial properties.
+4.5 The Chamber of Commerce reserves the right to revoke approval if the business deviates significantly from the approved proposal or violates any rules.
+
+### 5. Changes and Transfers
+5.1 Any significant changes to the business (e.g. change of location, major shift in products/services) must be submitted to the Chamber for approval.
+5.2 If an individual wishes to transfer ownership of their business to another individual, the new owner must submit a proposal to the Chamber for approval.
+
+### 6. Sim-Owned Businesses
+6.1 Businesses that are owned and operated by the sim administrators (i.e. not individually-owned) will be exempt from the application and approval process.
+6.2 These businesses will be carefully selected to enhance the overall roleplay experience and fill any gaps in the Woodcrest business ecosystem.
+6.3 Sim-owned businesses will still be required to adhere to all relevant rules and regulations.
+6.4 Sim-owned businesses may be managed ICly by members of the community by appointment from Sim Administrators. Sim Administrators reserve the right to change sim-owned businesses or their management at any time, for any reason. 
+6.5 Sim staff retain final say in the operation and usage of sim-owned businesses, including their use within storylines and other roleplay scenarios.
+
+### 7. Penalties for Non-Compliance
+7.1 Businesses that deviate from their approved proposal or violate any Woodcrest or North Carolina laws will face penalties, which may include:
+   a) Fines or civil penalties
+   b) Temporary closure or suspension of operations
+   c) Eviction from the commercial rental plot
+   d) In severe cases, forfeiture of any rent paid and permanent ban from operating a business in Woodcrest
+7.2 The specific penalties will be determined by the Chamber of Commerce and/or sim administrators based on the nature and severity of the violation.
+
+## Required Permits and Licenses
+The following types of businesses will require additional permits or licenses to operate in Woodcrest:
+- Restaurants and food service: Health department certification, food handler's permits for all employees
+- Bars and nightclubs: Alcohol permit from the North Carolina ABC Commission
+- Cannabis dispensaries: Dispensary license from the North Carolina Department of Health, proof of compliance with all state regulations
```

Comment:
What do we mean by:

> proof of compliance with all state regulations

Reply:
This is vague and I'm going to remove it.  I'll include a more direct way to reference this under Non-Compliance